docs/source/model_design/risk_factors.rst

```diff
@@ -134,8 +134,131 @@ Definitions
 Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+The **theoretical minimum risk exposure level (TMREL)** is the level of risk
+exposure that would minimize the risk of an adverse outcome for an individual.
+For example, the TMREL for smoking would be "has never smoked." The
+corresponding concept on the population level is the **theoretical minimum risk
+exposure distribution (TMRED)**, which is the distribution of risk exposure that
+would yield the lowest possible population risk. For smoking, the TMRED would be
+the trivial probability distribution assigning everyone in the population to the
+TMREL category "has never smoked." [WHO-Global-Health-Risks-Annex]_,
+[GBD-2017-Risk-Appendix]_
+
+Recall from the `causality section <What is a causal relationship?_>`_ that
+counterfactual analysis is used to describe the causal relationship between a
+risk factor and an outcome. **The TMRED is a particular choice of counterfactual
+exposure distribution** used for the causal attribution of disease burden to a
+given risk factor in a population (see `Population Attributable Fraction
+(PAF)`_). Other choices of counterfactual include the *plausible* minimum risk,
+*feasible* minimum risk, and *cost-effective* minimum risk, each of which can
+obviously depend on specific attributes of the population under consideration.
+On the other hand, Murray et al. state
+[Comparative-quantification-health-risks]_:
+
+  Biological principles as well as considerations of equity would necessitate
+  that, **although the exposure distribution for theoretical minimum risk may
+  depend on age and sex, it should in general be independent of geographical
+  region or population.**
+
+However, they go on to add:
+
+  Exceptions to this are however unavoidable. An example would be the case of
+  alcohol consumption, which in limited quantities and certain patterns, has
+  beneficial effects on cardiovascular mortality, but is always harmful for
+  other diseases such as cancers and accidents. In this case, the composition of
+  the causes of death as well as drinking patterns in a region would determine
+  the theoretical minimum distribution.
+
+.. todo::
+
+  Is there an updated/better example of TMRED depending on population, since the
+  latest research says that that there is no safe amount of alcohol?
```

Comment:
Seems you caught that.  Yes.  Iron deficiency.

Reply:
I think BMI is protective against some cancers as well.  You could make an argument that the TMRED/TMREL should be defined separately for different risk outcome pairs.  In fact, I think it's important to point out specifically that the TMREL is the level at which risk of **all** outcomes is minimized.

Reply:
Do you know of a reference that discusses iron deficiency?

What about a reference that says that TMREL minimizes risk for all outcomes?

Reply:
For BMI, it's mostly a thing I've heard in discussions with research people, though some literature talks about this: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830139/

The relationship is pretty complicated though cause there's reverse causality at work (e.g. chemo makes you lose weight).  Kiran has been working on the obesity washout stuff which is directly related and might have more info.

Reply:
Iron deficiency specifically has draw level TMREDs that vary by location.  I don't know what that means, but I think Ali looked into it some.

Reply:
I'm inferring the global nature of the TMREL from its operational definition.  The GBD only defines one TMREL/TMRED for the risk factor without consideration to the affected outcomes and apply it to all risk-outcome pairs in paf calculation.